app/assets/stylesheets/_base.scss

```diff
@@ -61,10 +61,12 @@ ul {
   }
 }
 
-
 a, button {
   &:focus {
     outline: 0;
   }
 }
 
+.flex {
```

Comment:
Foundation doesn't supply it but a lot of classes as align-center, etc. With flex class and foundation compliments I can position the elements keeping from writing another css only to that.